test/ir_York_test.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright 2023 Daniele Gobbetti
+
+#include "ir_York.h"
+#include "IRac.h"
+#include "IRrecv.h"
+#include "IRrecv_test.h"
+#include "IRsend.h"
+#include "IRsend_test.h"
+#include "gtest/gtest.h"
+
+
+// General housekeeping
+TEST(TestYork, Housekeeping) {
+  ASSERT_EQ("YORK", typeToString(decode_type_t::YORK));
```

Comment:
Also add a test for the other direction.
e.g.
```c++
  ASSERT_EQ(decode_type_t::YORK, strToDecodeType("YORK"));
```
You may have missed that code. Not sure.